rpg.py

```diff
@@ -1,12 +1,35 @@
+import render
+import util
+
 class Player:
     max_hp = 100
-    def __init__(self, name, gender, max_hp, health, fatigue, defence, strength, speed, intelligence):
+    def __init__(self, name, gender, level, xp, max_hp, health, fatigue, defence, strength, speed, intelligence):
         self.name = name
         self.gender = gender
+        self.level = level
+        self.xp = xp
         self.max_hp = max_hp
         self.health = health
         self.fatigue = fatigue
         self.defence = defence
         self.strength = strength
         self.speed = speed
-        self.intelligence = intelligence
\ No newline at end of file
+        self.intelligence = intelligence
+
+    def display_player_bar(self, level, xp, max_hp, health, fatigue):
+        # print xp/level
+        print("LV. " + str(util.calculate_current_level(self.xp)) + " (XP: " + str(self.xp) + "/" + str(round(util.xp_required_for_level(self.level))) + ")", end="")
+
+        # print health
+        print("\n" + "HP: ", end="")
+        render.print_bar(self.health, self.max_hp)
+        
+        # print fatigue
+        print("\n" + "FT: ", end="")
```

Comment:
```suggestion
        print()
        print("FT: ", end="")
```

Just to make it more readable

Reply:
Ok, I was doing it in the print to save space, I thought it was cleaner, but it doesn't read as well.